tests/integration_tests/Test/PM/Integration/CPFMerge.cls

```diff
@@ -0,0 +1,16 @@
+Class Test.PM.Integration.CPFMerge Extends Test.PM.Integration.Base
+{
+
+Parameter TargetModuleName As STRING = "cpf-merge";
+
+Method TestCPFMerge()
+{
+    Set tModuleDir = ..GetModuleDir(..#TargetModuleName)
+
+    Set tSC = ##class(%IPM.Main).Shell("load -v "_tModuleDir)
+    Do $$$AssertStatusOK(tSC, "Loaded module successfully")
+
+    // TODO - Add more tests here
```

Comment:
And assert that the contents of the CPF merge file were actually applied using other Config.* APIs in %SYS